build/postinstall.sh.cmake

```diff
@@ -201,9 +128,9 @@ extract_debug_symbols()
 mecho --blue "## Extracting debug info from files and saving them into dbg"
 
 # strip out debug symbols into external file
-case "$COMPILEFOR" in
-	apple)
-		echo "extraction of debug symbols for $COMPILEFOR currently not supported" >&2
+case "$SYSTEM_NAME" in
+	Apple)
+		echo "extraction of debug symbols for Apple currently not supported" >&2
 		i686-apple-darwin10-strip -S ${DESTDIR}bin/s25client
```

Comment:
Not sure. According to this: http://cmake.3232098.n2.nabble.com/Cross-compiling-a-static-library-CMake-2-8-3-td5907410.html CMake should automaticly detect ranlib and some other tools by the used compiliers. Best, if you'd check that on the build server first (carefull, ranlib is set manually in most cases) But for the sake of this PR leave it as is (no functional changes, using CMake objcopy is a new feature/bugfix independant of this, so let's keep it simple and get this one done first)